docs/ref/csrf.txt

```diff
@@ -150,12 +150,13 @@ Finally, you'll need to set the header on your AJAX request. Using the
 
     const request = new Request(
         /* URL */,
-        {headers: {'X-CSRFToken': csrftoken}}
+        {
+            headers: {'X-CSRFToken': csrftoken},
+            method: 'POST',
+            mode: 'same-origin', // Do not send CSRF token to another domain.
```

Comment:
As far as I'm aware we should chop trailing comma
```suggestion
            mode: 'same-origin' // Do not send CSRF token to another domain.
```

Reply:
I view the trailing comma as a matter of personal taste. I'm fine with either version.